### PR TITLE
add info to research.rst

### PR DESCRIPTION
**Pull request by tjboise**
add boise state university and a paper to the research.rst

### CODE REVIEW

File: docs/user/research.rst [hunk 1]
```diff
@@ -373,6 +374,7 @@ PINN
 #. Y\. Chen, L. Lu, G. Karniadakis, & L. Negro. `Physics-informed neural networks for inverse problems in nano-optics and metamaterials <https://doi.org/10.1364/OE.384875>`_. *Optics Express*, 28(8), 11618--11633, 2020.
 #. G\. Pang, L. Lu, & G. Karniadakis. `fPINNs: Fractional physics-informed neural networks <https://doi.org/10.1137/18M1229845>`_. *SIAM Journal on Scientific Computing*, 41(4), A2603--A2626, 2019.
 #. D\. Zhang, L. Lu, L. Guo, & G. Karniadakis. `Quantifying total uncertainty in physics-informed neural networks for solving forward and inverse stochastic problems <https://doi.org/10.1016/j.jcp.2019.07.048>`_. *Journal of Computational Physics*, 397, 108850, 2019.
+#. T\. Zhang, D. Wang, & Y. Lu. `RheologyNet: A physics-informed neural network solution to evaluate the thixotropic properties of cementitious materials <https://doi.org/10.1016/j.cemconres.2023.107157>`_. *Cement and Concrete Research, 168, 107157, 2023.
```
Comment (lululxvi) [inline docs/user/research.rst:377]: Order by year.

Comment (tjboise) [reply docs/user/research.rst]: updated

File: docs/user/research.rst [hunk 2]
```diff
@@ -211,6 +212,7 @@ PINN
 #. J\. Li, Y. Lin, & K. Zhang. `Dynamic mode decomposition of the core surface flow inverted from geomagnetic field models <https://agupubs.onlinelibrary.wiley.com/doi/abs/10.1029/2023GL106362>`_. *Geophysical Research Letters*, 51(1), e2023GL106362, 2024.
 #. J\. M. Tucny, M. Durve, A. Montessori, and S. Succi. `Learning of viscosity functions in rarefied gas flows with physics-informed neural networks <https://www.sciencedirect.com/science/article/pii/S0045793023003390>`_. *Computers Fluids*, 269:106114, 2024.
 #. P\. Brendel, V. Medvedev, & A. Rosskopf. `Physics-informed neural networks for magnetostatic problems on axisymmetric transformer geometries <https://ieeexplore.ieee.org/abstract/document/10373037>`_. *IEEE Journal of Emerging and Selected Topics in Industrial Electronics*, 2023.
+#. T\. Zhang, D. Wang, & Y. Lu. `RheologyNet: A physics-informed neural network solution to evaluate the thixotropic properties of cementitious materials <https://doi.org/10.1016/j.cemconres.2023.107157>`_. *Cement and Concrete Research, 168, 107157, 2023.
```
Comment (lululxvi) [inline docs/user/research.rst:215]:
```
*Cement and Concrete Research*
```

File: docs/user/research.rst [hunk 3]
```diff
@@ -153,7 +153,8 @@ DeepXDE has been used in
   `National University of Defence Technology <https://english.nudt.edu.cn/About/index.htm>`_,
   `University of Applied Sciences and Arts Northwestern Switzerland <https://www.fhnw.ch/en/startseiteu>`_,
   `University of Los Andes <https://uniandes.edu.co/en>`_,
-  `Kuwait University <https://kuweb.ku.edu.kw>`_
+  `Kuwait University <https://kuweb.ku.edu.kw>`_,
+  `Boise State University <https://www.boisestate.edu/>`_
```
Comment (lululxvi) [inline docs/user/research.rst:157]: Delete the last `/`

Comment (tjboise) [reply docs/user/research.rst]: have uploaded the revised one

**Pull request merged**